internal/store/job_test.go

```diff
@@ -214,6 +214,30 @@ func TestJobStore(t *testing.T) {
 				kube_job_status_failed{job_name="FailedJob1",namespace="ns1",reason="Evicted"} 0
 				kube_job_status_start_time{job_name="FailedJob1",namespace="ns1"} 1.495807207e+09
 				kube_job_status_succeeded{job_name="FailedJob1",namespace="ns1"} 0
+`,
+		},
+		{
+			Obj: &v1batch.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "FailedJobWithNoConditions",
+					Namespace: "ns1",
+				},
+				Status: v1batch.JobStatus{
+					Failed: 1,
+				},
+				Spec: v1batch.JobSpec{
+					ActiveDeadlineSeconds: &ActiveDeadlineSeconds900,
+				},
+			},
+			Want: metadata + `
+				kube_job_annotations{job_name="FailedJobWithNoConditions",namespace="ns1"} 1
```

Comment:
```suggestion

```

Reply:
Ahh I just needed to sync to HEAD and it started failing locally too. Thanks fixed now.